easybuild/easyconfigs/n/naturalsort/naturalsort-0.1.3-foss-2020b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'RPackage'
+
+name = 'naturalsort'
```

Comment:
This easyconfig file should be removed, since we're adding `naturalsort` as an extension to R, see #13762

Reply:
done